app/views/course/assessment/submission/submissions/_questions.json.jbuilder

```diff
@@ -18,6 +18,7 @@ json.questions question_assessments.each_with_index.to_a do |(question_assessmen
   submission_question = sq_topic_ids_hash[question.id][0]
   json.partial! 'question', question: question, can_grade: can_grade, answer: answer
   json.displayTitle question_assessment.display_title(index + 1)
```

Comment:
If you intend to just pass the question number, why not just get the question_number right away? using `display_title` is misleading as it is not returning display_title but just the number?

Reply:
understand about this part, and also understand that actually they have defined `default_title` as such (`Question %{index}`), so I use this instead